packages/core/integration-tests/test/integration/react-refresh-library-target/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "source": "index.js",
+  "main": "dist/main.js",
+  "dependencies": {
+    "@parcel/transformer-react-refresh-wrap": "*",
```

Comment:
Is this dependency needed?

Reply:
oh, probably not. This is probably from when I was experimenting with different package configs. I'll trim this down.

Reply:
Actually, I think it is; if this is removed, forcing the the test to fail (by commenting out the fix) doesn't manifest because `External dependency "@parcel/transformer-react-refresh-wrap" is not declared in package.json.`